presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java

```diff
@@ -314,7 +360,7 @@ protected PlanNodeStatsEstimate visitInPredicate(InPredicate node, Void context)
 
             InListExpression inList = (InListExpression) node.getValueList();
             ImmutableList<PlanNodeStatsEstimate> equalityEstimates = inList.getValues().stream()
-                    .map(inValue -> process(new ComparisonExpression(EQUAL, node.getValue(), inValue)))
+                    .map(inValue -> process(new ComparisonExpression(ComparisonExpression.Operator.EQUAL, node.getValue(), inValue)))
```

Comment:
Is this intended? It should be fine without.

Reply:
I guess it's due to auto format. Will fix.